sam-hiberry-app/order_processing/app.py

```diff
@@ -0,0 +1,123 @@
+import json
+import logging
+import os
+import requests
+
+# Initialize logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Shopify identifier
+SHOPIFY_ID = 0
+
+CREATE_ORDER_API_URL = os.getenv('CREATE_ORDER_API_URL', None)
```

Comment:
if this is not set, it will fail

Reply:
Raising ConfigurationError instead . In theory it shouldn't fail on AWS, as it is injected by sam.